docs/root/configuration/cluster_manager/cluster_hc.rst

```diff
@@ -14,52 +14,26 @@ Health checking
 TCP health checking
 -------------------
 
-.. attention::
-
-  This section is written for the v1 API but the concepts also apply to the v2 API. It will be
-  rewritten to target the v2 API in a future release.
-
-The type of matching performed is the following (this is the MongoDB health check request and
-response):
+The type of matching performed is the following:
 
 .. code-block:: json
 
   {
     "send": [
-      {"binary": "39000000"},
-      {"binary": "EEEEEEEE"},
-      {"binary": "00000000"},
-      {"binary": "d4070000"},
-      {"binary": "00000000"},
-      {"binary": "746573742e"},
-      {"binary": "24636d6400"},
-      {"binary": "00000000"},
-      {"binary": "FFFFFFFF"},
-      {"binary": "13000000"},
-      {"binary": "01"},
-      {"binary": "70696e6700"},
-      {"binary": "000000000000f03f"},
-      {"binary": "00"}
+      {"text": "39000000"},
+      {"text": "EEEEEEEE"},
```

Comment:
How come you removed the full example?

Reply:
full example was binary, and binary is hidden in v2 API.  I figured it wasn't really relevant without base64 converting which I didn't want to bother with.

Reply:
As mentioned below, I think we need to consistently move to YAML. Also, I think this example is wrong for v2, see https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/core/health_check.proto#L118. `send` is now a single `Payload`.